### DISCUSSION TITLE
Using Merged NFS mounts points for writing

### DISCUSSION THREAD

**Discussion by gpradeepjain** [trapexit/mergerfs#942] [General]
Hi, We have been using MergerFS over 2 NFS mounts each 100TB. The merged folder was always used for reading purposes and we had great success using this. 

Will, there be any problem if we use the merged path over NFS for writes as well.

What will happen in the below scenario?

NFS1 -> folderA
NFS2 -> folderA

Merged path over NFS NFS3 -> Will have merged folderA

If user writes to NFS3 -> folderA where will the new files be written into?

**Comment by trapexit** [comment]
Files are created based on the [policy ](https://github.com/trapexit/mergerfs#functions-categories-and-policies) you choose and the metrics at the time.

**Reply by gpradeepjain** [reply]
Sure, let me read on that. If I use it for writes using NFS like i explained above, so you foresee any issues?

**Comment by trapexit** [comment]
The fact it is NFS is only relevant to the situations where NFS is non-POSIX compliant which is generally documented in the mergerfs docs such as: https://github.com/trapexit/mergerfs#nfsopenhack

Otherwise it's the same as any other filesystem.
